sreftml/plots.py

```diff
@@ -564,8 +558,8 @@ def residual_plot(
     n_biomarker = len(name_biomarkers)
     n_row, n_col = n2mfrow(n_biomarker, ncol_max)
     x_data = df.TIME.values + df.offsetT.values
-    y_pred = scaler_y.inverse_transform(sreft(scaled_features))
-    y_res = y_pred - df[name_biomarkers].values
+
+    y_res = df[[f"{biomarker}_pred" for biomarker in name_biomarkers]].values - df[name_biomarkers].values
```

Comment:
How about a warning prompting to use calculate_offsetT_prediction when offsetT or pred is not available?

Reply:
Good idea, I'll implement it.